peak_finder/src/navigator.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 RoboJackets
```

Comment:
Why hpp is no implementation in this file? What is the dividing line between hpp and h?

Reply:
A lot of the ROS 2 stuff seems to be following *.h for C, *.hpp for C++, and I've just kinda been rolling with it.